src/java.base/share/classes/java/time/ZoneOffset.java

```diff
@@ -714,7 +714,9 @@ public Temporal adjustInto(Temporal temporal) {
      * The comparison is "consistent with equals", as defined by {@link Comparable}.
      *
      * @param other  the other date to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value is less than zero if the {@code other} is before,
```

Comment:
It is tricky to describe an offset as before or after. If you are going to change this you will need a much better description